### PR TITLE
improves readme

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,39 +2,48 @@
...
 
-operatorkit package is a library for creating [Kubernetes
-operators][operators]. It emerged as we extracted common functionality from
-number of the operators we developed at Giant Swarm. The goal of the library is
-to provide a common structure of operator projects and to encapsulate best
-practices learned while running operators in production.
+Package `operatorkit` implements an opinionated framework for developing
+[Kubernetes operators][operators]. It emerged as we extracted common
+functionality from a number of the operators we developed at [Giant
+Swarm][giantswarm]. The goal of this library is to provide a common structure of
```
Comment (rossf7) [inline README.md]: for operator

File: README.md [hunk 2]
```diff
@@ -2,39 +2,48 @@
...
-  library.
-- Possible to change behaviour that often is specific to an organization
-  like logging and error handling.
+- CRD primitives to reliably create, watch and delete custom resources, as well
+  as any Kubernetes runtime object.
+- Managing [finalizers][finalizers] on reconciled objects, making sure the code
+  is executed at least once for each create/delete/update event.
+- A deterministic informer implementation that guarantees the expected behaviour
+  of configured resync periods and rate waits.
+- Convenient client library helpers for more simple client creation.
```
Comment (rossf7) [inline README.md]: simpler client creation

File: README.md [hunk 3]
```diff
@@ -2,39 +2,48 @@
...
+- Resource wrapping to gain ability of composing resources like middlewares.
+- Control flow primitives that allow cancelation and repetition of resource
+  implementations.
+- Independent packages. It is possible to use only certain parts of the library
+  without being bound to all primitives it provides.
+- Possible to change behaviour that is often specific to an organization like
```
Comment (rossf7) [inline README.md]: Ability to change

File: README.md [hunk 4]
```diff
@@ -2,39 +2,48 @@
...
   dispatch events twice after the resync period, which saves some cycles. It
   also features rate limiting of the event dispatching. It also provides
   functionality for decoding custom objects, reducing error prone boilerplate
   code.
-- controller - provides a framework aiming to help writing reliable, robust
-  controllers performing reconciliation loops. The heart of the controller is
-  a Resource interface. The reconciliation primitive allowing split the
+- `controller`: provides a framework aiming to help writing reliable, robust
+  controllers performing reconciliation loops. The heart of the controller is a
+  Resource interface. The reconciliation primitive allowing split the
```
Comment (rossf7) [inline README.md]: allows splitting

File: README.md [hunk 5]
```diff
@@ -2,39 +2,48 @@
...
   reconciliation into smaller bits. Controller manages [finalizers][finalizers]
```
Comment (rossf7) [inline README.md]: smaller parts

**Comment by xh3b4sd** [comment]
Cool, thanks for the feedback. Fixed. Going.

**Pull request merged**